tutorials/clinical/30_ecog.py

```diff
@@ -134,6 +135,31 @@
     envelope=True).resample(sfreq)
 gamma_info = gamma_power_t.info
 
+# %%
+# Plot Gamma Power on cortical sensors
+# --------------------------------------
+#
+# We will now use evoked gamma power to plot on the cortical surface.
+# Therefore we extract the evoked time sample at 15s and normalize
+# it in a range of 0 to 1 in order to map it using a matplotlib colormap.
```

Comment:
I like this idea!  But the tutorial is not building correctly on our CircleCI server. Does it work locally for you to run the tutorial?

Reply:
Yes, it does. It plots the spheres as expected:
![image](https://user-images.githubusercontent.com/38216460/198151228-a9bfd0d4-73cb-4474-8738-cdd73ed27784.png)

Seeing that now I think the adding a colorbar added might be also useful.